src/plugins/data/public/index.ts

```diff
@@ -36,3 +36,16 @@ export * from './types';
 export { IRequestTypesMap, IResponseTypesMap } from './search';
 export * from './search';
 export * from './query';
+export {
```

Comment:
We want to have an `indexPattern` name space like with `esFilters` 

Then things like `IndexPatternMissingIndices` will become `MissingIndices`, `INDEX_PATTERN_ILLEGAL_CHARACTERS_VISIBLE` will become `ILLEGAL_CHARACTERS_VISIBLE` etc.

Reply:
Done